README.md

```diff
@@ -87,7 +87,7 @@ K9s is available on Linux, macOS and Windows platforms.
 
   ```shell
   # NOTE: The dev version will be in effect!
-  go get -u github.com/derailed/k9s
+  go install github.com/derailed/k9s
```

Comment:
@kevingentile Thank you for this! I think we should specify the command as so to indicate you can pull a diff rev if you want to... ie 

```shell
go install github.com/derailed/k9s@latest
```